torch/_inductor/codegen/wrapper.py

```diff
@@ -192,7 +192,11 @@ def determine_grid(
 
     def writeline(line: str, example_grid: Optional[str] = None):
         output.writeline(line)
-        if wrapper and config.triton.autotune_at_compile_time:
+        if (
+            wrapper
+            and config.triton.autotune_at_compile_time
+            and name not in wrapper.kernel_autotune_names
+        ):
             wrapper.kernel_autotune_calls.writeline(example_grid or line)
```

Comment:
This check fixes the case when we call the same UDF Triton kernel. This will prevent the same grid fn from being codegen-ed multiple times in the standalone autotune file.

```
    def grid_wrapper_for_mul2_inplace_kernel_0(meta):
        return (4, 1, 1)
    arg0_1 = generate_example_value((4, 4), (4, 1), 'cuda:0', torch.float32, 0)
    mul2_inplace_kernel_0.run(arg0_1, 4, 16, grid=grid_wrapper_for_mul2_inplace_kernel_0, stream=stream0)
    del arg0_1
    def grid_wrapper_for_mul2_inplace_kernel_0(meta):
        return (4, 1, 1)
    def grid_wrapper_for_mul2_inplace_kernel_0(meta):
        return (4, 1, 1)
```